README.md

```diff
@@ -393,6 +393,8 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Gemfury](https://gemfury.com) — Private and public artifact repos for Maven, PyPi, NPM, Go Module, Nuget, APT, and RPM repositories. Free for public projects.
   * [paperspace](https://www.paperspace.com/) — Build & scale AI models, Develop, train, and deploy AI applications, free plan: public projects, 5Gb storage, basic instances.
   * [RepoForge](https://repoforge.io) - Private cloud-hosted repository for Python, Debian, NPM packages and Docker registries. Free plan for open source/public projects.
+  * [RepoFlow](https://repoflow.io) - RepoFlow simplifies package management with support for npm, PyPI, Docker, Go, Helm, and more. Try it for free, in the cloud or self-hosted.
```

Comment:
please include details of what is free, do not add new blank lines, thank you!

Reply:
No problem, I will. Do you mean like explain the limits of the free plan?

Reply:
yes, you can see many examples here of other services doing that, so just do the same

Reply:
I don’t have any restrictions, such as “or open source/public projects.” It’s completely free for everyone, but I’ll do my best to ensure it qualifies.

Reply:
I’ve added everything